docs/file-events.yml

```diff
@@ -1244,3 +1283,338 @@ components:
     StreamingResponseBody:
       title: StreamingResponseBody
       type: object
+    SavedSearchResponse-file-events:
+      title: SavedSearchResponse
+      type: object
+      properties:
+        searches:
+          type: array
+          description: List of saved searches in the response.
+          items:
+            $ref: '#/components/schemas/SavedSearchDetail-file-events'
+    SavedSearchDetail-file-events:
```

Comment:
This is all mostly just copied over from the downloaded file from the dev portal project - saved search was added after the initial mock servers creation for file events